math-scala/src/main/scala/org/apache/mahout/math/algorithms/regression/LinearRegressorModel.scala

```diff
@@ -123,9 +156,33 @@ trait LinearRegressorFitter[K] extends RegressorFitter[K] {
   }
 
   def generateSummaryString[M[K] <: LinearRegressorModel[K]](model: M[K]): String = {
+
+    /*  Model after R implementation ...
+    Call:
+      lm(formula = target ~ a + b + c + d, data = df1)
+
+    Residuals:
+    1       2       3       4       5       6       7       8       9
+    -4.2799  0.5059 -2.2783  4.3765 -1.3455  0.7202 -1.8063  1.2889  2.8184
+
+    Coefficients:
+      Estimate Std. Error t value Pr(>|t|)
+    (Intercept)  163.179     51.915   3.143   0.0347 *
```

Comment:
Not yet, I was going to explore moving the intercept to the first index in the array for seperate PR.